cmd/dusk/server.go

```diff
@@ -100,7 +103,28 @@ func Setup() *Server {
 	// creating the rpcbus
 	rpcBus := rpcbus.New()
 
-	counter := chainsync.NewCounter()
+	_, db := heavy.CreateDBConnection()
+	processor := peer.NewMessageProcessor(eventBus)
+	// Register peer services
+	processor.Register(topics.Ping, responding.ProcessPing)
+	dataBroker := responding.NewDataBroker(db, rpcBus)
+	processor.Register(topics.GetData, dataBroker.SendItems)
+	dataRequestor := responding.NewDataRequestor(db, rpcBus, eventBus)
+	processor.Register(topics.Inv, dataRequestor.RequestMissingItems)
+	bhb := responding.NewBlockHashBroker(db)
+	processor.Register(topics.GetBlocks, bhb.AdvertiseMissingBlocks)
+	cb := responding.NewCandidateBroker(db)
+	processor.Register(topics.GetCandidate, cb.ProvideCandidate)
```

Comment:
Is any component gossiping `topics.Candidate` anywhere?  As per protocol, we fetch the candidate from network, if it cannot be retrieved from local state.

Reply:
Ah, I left the code commented. I can add it back to the candidate broker.